third_party/move/move-prover/boogie-backend/src/options.rs

```diff
@@ -20,8 +20,13 @@ const DEFAULT_BOOGIE_FLAGS: &[&str] = &[
 ];
 
 const MIN_BOOGIE_VERSION: &str = "2.15.8";
+const MAX_BOOGIE_VERSION: &str = "2.15.8";
+
 const MIN_Z3_VERSION: &str = "4.11.0";
+const MAX_Z3_VERSION: &str = "4.11.0";
+
 const MIN_CVC5_VERSION: &str = "0.0.3";
+const MAX_CVC5_VERSION: &str = "0.0.3";
```

Comment:
This one caused an linter error: https://github.com/aptos-labs/aptos-core/actions/runs/5789663535/job/15691166679 You could remove it since it is not used.

Reply:
Removed it. Thank you.